sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/ReceiveLinkHandler.java

```diff
@@ -22,15 +21,12 @@
 public class ReceiveLinkHandler extends LinkHandler {
     private final String linkName;
     private final AtomicBoolean isFirstResponse = new AtomicBoolean(true);
-    private final DirectProcessor<Delivery> deliveries;
-    private final FluxSink<Delivery> deliverySink;
+    private final Sinks.Many<Delivery> deliveries = Sinks.many().multicast().onBackpressureBuffer();
```

Comment:
Based on https://projectreactor.io/docs/core/release/api/reactor/core/publisher/DirectProcessor.html the closest Sinks equivalent would be `Sinks.many().multicast().directBestEffort()`